### ISSUE TITLE
Lint check fails with "remote not found" error

### ISSUE THREAD

**Issue by amkozlov** [bioconda/bioconda-recipes#24592]
My PR #24522 fails in the lint step with the same error as reported in #19462 :

```
  File "/home/circleci/project/miniconda/lib/python3.7/site-packages/bioconda_utils/githandler.py", line 143, in get_remote
    raise KeyError(f"No remote matching '{desc}' found")
KeyError: "No remote matching 'bioconda/bioconda-recipes' found"
```

Except in my case it's not a local build, so I cannot use the workaround suggested in #19462

Any help much appreciated!

**Comment by amkozlov** [comment]
For whatever reason, CircleCI is running checks on the `bump/raxml-ng` branch in my fork of `bioconda-recipes`:

![image](https://user-images.githubusercontent.com/5624530/94837131-22860e80-0414-11eb-8466-afb14f61f54a.png)

whereas for other PRs that completed the checks successfully, it created a special `pull/xxxx` branch:

![image](https://user-images.githubusercontent.com/5624530/94837507-9de7c000-0414-11eb-8c96-6a8c02c202f0.png)

Why is this happening and how can I fix this?

**Comment by AngieHinrichs** [comment]
I am having the same issue: I forked bioconda-recipes, made a branch on my laptop, made a small change, pushed my branch to my fork on github, and then CircleCI lint [fails](https://app.circleci.com/pipelines/github/AngieHinrichs/bioconda-recipes/3/workflows/08c80246-a10d-43f1-9a03-dfba84a61bd2/jobs/4) with
```
  File "/home/circleci/project/miniconda/lib/python3.7/site-packages/bioconda_utils/githandler.py", line 143, in get_remote
    raise KeyError(f"No remote matching '{desc}' found")
KeyError: "No remote matching 'bioconda/bioconda-recipes' found"
```

I have not been able to find a way to add or change a remote for my github repo in the web interface (as opposed to command line on my laptop).

@amkozlov were you able to get around this somehow?

**Comment by amkozlov** [comment]
@AngieHinrichs well my "workaround" was asking a colleague to re-submit the PR from his fork. It worked, so probably something was broken in my repo/fork, but I have no idea what exactly.

**Comment by AngieHinrichs** [comment]
I asked for help in the gitter channel, and @dpryan79 asked that I disable CircleCI for my fork of bioconda-recipes because it was somehow interfering with checks on my PR to the main repo.  So I guess CircleCI is not supposed to work on forks?

**Comment by dpryan79** [comment]
Correct, if you have circleci enabled on a fork, then it will not run for the main repository. I presume this prevents it from running twice for no reason, but I don't really know.
